src/PHPCensor/Migrations/20170321131931_add_environment.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddEnvironment extends AbstractMigration
+{
+    public function up()
+    {
+        $table = $this->table('environment');
+
+        if (!$this->hasTable('environment')) {
+            $table->create();
+        }
+
+        if (!$table->hasColumn('project_id')) {
+            $table->addColumn('project_id', 'integer')->save();
+        }
+
+        if (!$table->hasColumn('name')) {
+            $table->addColumn('name', 'string', ['limit' => 20])->save();
```

Comment:
I would make a limit of 250 (like a 'branch' field in the 'build' table). 20 is too short